js/components/MapView/index.js

```diff
@@ -0,0 +1,5 @@
+import MapView from "./MapView";
+import MapViewContainer from "./MapViewContainer";
```

Comment:
Components do not have container files. Please remove MapViewContainer file. If it's a screen then move it to the screens dir.